src/mapping/mapping_main.py

```diff
@@ -96,6 +96,14 @@ def run_mapping(
     else:
         MappingMainLogger.info(f"Reference list not updated for survey year {year}.")
 
+    # Add custom mappers if PNP is true for later imp_class column creation
+    if config["survey"]["survey_type"] == "PNP":
+        full_responses = hlp.identify_osmotherly_key_area(
```

Comment:
pretty good, and I like having all of this together in one place.
Just a couple of suggestions, can the new scripts you added to mapping_helpers be put in another separate script called pnp_mapping.py or something? Just like we did for cellno, itl, pg, ultfoc. Don't spend too long on it though, as we're not 100% sure exactly how we'll be doing this.

Reply:
Done :)